src/components/button/button-classic-config.style.js

```diff
@@ -0,0 +1,225 @@
+export default {
+  primary: {
+    blue: `
+      background: #255bc7;
+      border: 1px solid transparent;
+      color: #ffffff;
+      &:hover {
+        background: #1e499f;
+        border: 1px solid #1e499f;
+        color: #ffffff;
+      }
+      &:focus {
+        box-shadow: 0 0 6px rgba(25,99,246,.6);
+        outline: none;
+      }
+    `,
+    grey: `
+      background: #335c6d;
+      border: 1px solid transparent;
+      color: #ffffff;
+      &:hover {
+        background: #003349;
+        border: 1px solid #003349;
+        color: #ffffff;
+      }
+      &:focus {
+        box-shadow: 0 0 6px rgba(51,92,109,.6);
+        outline: none;
+      }
+    `,
+    magenta: `
+      background: #ed1c5f;
+      border: 1px solid transparent;
+      color: #ffffff;
+      &:hover {
+        background: #be164c;
+        border: 1px solid #be164c;
+        color: #ffffff;
+      }
+      &:focus {
+        box-shadow: 0 0 6px rgba(243,46,109,.6);
+        outline: none;
+      }
+    `,
+    'magenta-dull': `
+      background: #ca2a60;
+      border: 1px solid transparent;
+      color: #ffffff;
+      &:hover {
+        background: #a2224a;
+        border: 1px solid #A2224A;
+        color: #ffffff;
+      }
+      &:focus {
+        box-shadow: 0 0 6px rgba(213,85,128,.6);
+        outline: none;
+      }
+    `,
+    red: `
+      background: #c7384f;
+      border: 1px solid transparent;
+      color: #ffffff;
+      &:hover {
+        background: #9f2d3f;
+        border: 1px solid #9f2d3f;
+        color: #ffffff;
+      }
+      &:focus {
+        box-shadow: 0 0 6px rgba(228,44,45,.6);
+        outline: none;
+      }
+    `,
+    white: `
+      background: #ffffff;
+      border: 1px solid transparent;
+      color: rgba(0,0,0,85);
+      &:hover {
+        background: #ccc;
+        border: 1px solid #ccc;
+        color: #1b1d21;
+      }
+      &:focus {
+        box-shadow: 0 0 6px rgba(51,92,109,.6);
+        outline: none;
+      }
+    `
+  },
+  secondary: {
+    blue: `
+      background: transparent;
+      border: 1px solid #255bc7;
```

Comment:
You could declare variables for some of these repeated colours.

Reply:
Good point, as the classic stuff is likely/ hopefully going to be fazed out down the line I've defined them in the same file